src/EditorFeatures/Core/Implementation/Formatting/Indentation/AbstractSmartTokenFormatterCommandHandler.cs

```diff
@@ -23,6 +23,8 @@
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.Formatting.Indentation
 {
+    using Microsoft.CodeAnalysis.Indentation;
```

Comment:
same here?

Reply:
can't.  if we move it, then diffrent symbols will be found in Microsoft.CodeAnalysis.Editor.